src/components/Personalization/createOnDecisionHandler.js

```diff
@@ -30,13 +30,19 @@ export default ({
     const { render, returnedPropositions } = processPropositions(
       propositionsToExecute,
     );
+    debugger;
 
     const handleNotifications = notificationHandler(
       renderDecisions,
       sendDisplayEvent,
       viewName,
     );
-    render().then(handleNotifications);
+    render().then(
+      handleNotifications.bind(
+        null,
+        returnedPropositions.filter((p) => p.isSuppressedDisplay),
```

Comment:
I think these params are in the wrong order. See createNotificationHandler.js#L28